sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -41,6 +40,22 @@ export interface TokenResponse {
   refreshToken?: string;
 }
 
+/**
+ * @internal
+ */
+export function getIdentityClientAuthorityHost(options?: TokenCredentialOptions): string {
+  // The authorityHost can come from options or from the AZURE_AUTHORITY_HOST environment variable.
+  let authorityHost = options?.authorityHost;
+
+  // The AZURE_AUTHORITY_HOST environment variable can only be provided in NodeJS.
+  if (isNode) {
+    authorityHost = authorityHost || process.env.AZURE_AUTHORITY_HOST;
```

Comment:
It's slightly better for bundlers if we use a `.browser.ts` file and a browser-mapping instead of using `isNode`. If we do it this way, bundlers can complain about having no `process` global. That said, we do this in many other places in identity already, so it's probably not a big deal.

Reply:
We’ll have to clean this client soon, let’s leave that for that cleanup.